datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/translate_profile.py

```diff
@@ -37,7 +37,7 @@ def fetch_mib(mib, source_url):
 @click.argument('profile_path')
 @click.option(
     '--mib_source_url',
-    default='https://raw.githubusercontent.com/projx/snmp-mibs/master/@mib@',
+    default='https://raw.githubusercontent.com:443/DataDog/mibs.snmplabs.com/master/asn1/@mib@',
```

Comment:
Could you also update https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/generate_profile.py#L26

Or better, create a constant used at both location.